src/threaded_rodeo.rs

```diff
@@ -979,8 +979,9 @@ where
             .expect("failed to allocate memory for interner");
 
         for (string, key) in deser_map {
-            if key.into_usize() > highest {
-                highest = key.into_usize();
+            let next_key = key.into_usize() + 1;
```

Comment:
Is this panicking/wrapping during deserialization possible?

Reply:
You mean if `next_key` overflows? That should only happen if the `Rodeo` is completely filled. I can do a saturating add though.